docs/utilities/baseline-grid.md

```diff
@@ -12,3 +12,9 @@ You can apply this utility to an element (such as `<body>`) to give it a series
     class="js-example">
 View example of the baseline grid utility
 </a>
+
+<div class="p-notification--caution">
+  <p class="p-notification__response">
+    <span class="p-notification__status">Warning:</span>This pattern requires JS to be functional.
+  </p>
+</div>
```

Comment:
Not required here as the grid lines are not part of the framework itself.

Reply:
Removed 👍